### PR TITLE
WIP BXMSDOC-5893: Made additions and modifications as per BAPL-1416,1585 and 1393

### PR DESCRIPTION
**Pull request by hmanwani-rh**
See JIRA: https://issues.redhat.com/browse/BXMSDOC-5893
Epic: https://issues.redhat.com/browse/BXMSDOC-5605

Rendered output:
[Designing a decision service using guided decision tables RHPAM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-5893-RHPAM/)
[Designing a decision service using guided decision tables RHDM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-5893-RHDM/)

Community versions
[Drools](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-5893-DROOLS/#guided-decision-tables-con)

### CODE REVIEW

**Review by Rikkola — APPROVED** [review]

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-convert-proc.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ ifdef::DROOLS,JBPM,OP[]
 xref:decision-tables-con_decision-tables[].
 endif::[]
 
-WARNING: Guided decision tables and spreadsheet decision tables are different decision table formats that support different features. Any supported features that differ between the two decision table formats are modified or lost when you convert one decision table format to the other.
+WARNING: Guided decision tables and spreadsheet decision tables are different decision table formats that support different features. Any supported features that differ between the two decision table formats (Hit policy, for example) are modified or lost when you convert one decision table format to the other.
```
Comment (michelehaglund) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-convert-proc.adoc]: Looks good. Minor comments.

A lot of instances of "decision". I think that one can be removed. 

WARNING: Guided decision tables and spreadsheet decision tables are different table formats that support different features. Supported features that differ between the two decision table formats (Hit policy, for example) are modified or lost when you convert one decision table format to the other.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[id='guided-decision-tables-rulename-column-view-proc']
+= Viewing Rule Name column in guided decision tables
+
+You can view the *Rule Name* column in the guided decision table if needed.
+
+.Procedure
+. In the guided decision tables designer, click *Columns*.
+. Select *Show rule name column* check box.
```
Comment (michelehaglund) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc]: Select "the"..

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+. Click *Finish* in the wizard to save.
```
Comment (michelehaglund) [inline doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc]: Click *Finish* to save your changes.

Comment (michelehaglund) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc]: Or simply, "Click *Finish*.

**Review by michelehaglund — APPROVED** [review]
Looks good @hmanwani-rh Minor comments.

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-convert-proc.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-convert-proc.adoc]: Done

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc (hunk 2 above)
Comment (hmanwani-rh) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc]: Done

File: doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc (hunk 3 above)
Comment (hmanwani-rh) [reply doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/guided-decision-tables-rulename-column-view-proc.adoc]: changed to Click Finish to save.

**Pull request merged**